### DISCUSSION TITLE
Asistencia de los lives

### DISCUSSION THREAD

**Discussion by JoseSomilledaLaunchX** [Launch-X-Latam/MisionFrontEnd#67] [Q&A]
Se mencionaba que se tenian que asistir al menos a 5 sesiones. Como se evalua la asistencia?

**Comment by JosafatJimenezB** [comment]
Como tal no se evalua, sino que teams detecta si te conectas o no asi que te recomiendo entrar cuando puedes para que detecte que estas activo, para que vean que participas
